Gemfile

```diff
@@ -233,7 +233,12 @@ def override_gem(name, *args)
 
     calling_file = caller_locations.detect { |loc| !loc.path.include?("lib/bundler") }.path
     gem(name, *args).tap do
-      warn "** override_gem: #{name}, #{args.inspect}, caller: #{calling_file}" unless ENV["RAILS_ENV"] == "production"
+      verbose = if ENV["GEMFILE_SILENT"]
+                  %w(false 0).include?(ENV["GEMFILE_SILENT"])
```

Comment:
Huh? `GEMFILE_SILENT=0` is verbose? The naming of this variable and/or the fact that I'm explicitly putting 0 or false to enable a non-default mode is very confusing to me.